game-app/game-core/src/main/resources/i18n/games/strategy/engine/framework/ui_en.properties

```diff
@@ -7,7 +7,7 @@ startup.MainPanel.btn.Play.Tltp.line3=Play Online, or Local Game, or PBEM, or Ho
 startup.MainPanel.btn.Quit.Lbl=Quit
 startup.MainPanel.btn.Quit.Tltp=Close TripleA.
 startup.PbfSetupPanel.tab.ForumPosterEditor.Lbl=Play By Forum
-startup.PlayerSelectorRow.btn.Alliances.Tltp=Click to play {0}
+startup.PlayerSelectorRow.btn.Alliances.Tltp=Click to play %s
```

Comment:
AFAIK, internationalization is actually a place where `MessageFormat` has a difference. Are we sure this does not break any of the NLS work?